pkg/asset/tls/certkey.go

```diff
@@ -14,6 +14,16 @@ import (
 	"github.com/openshift/installer/pkg/types"
 )
 
+const (
+	// KeyIndex is the index into a CertKey asset's contents at which the key
+	// can be found.
+	KeyIndex = 0
+
+	// CertIndex is the index into a CertKey asset's contents at which the
+	// certificate can be found.
+	CertIndex = 1
```

Comment:
This is fine for now, but it makes me think the asset content should be a filename->content map instead of a slice.

Reply:
After this merges, I want to revisit our asset store. It's got some room for improvement.